<launch>
  <rosparam command="load" file="$(find aravis_camera_ros_driver)/config/camera_params.yaml" />
  <node 
    pkg="aravis_camera_ros_driver"
    type="aravis_camera_node"
    name="aravis_camera_node"
    respawn="true"
    output="screen"
  >
  </node>

</launch>
